datafusion/tests/data_test_context/data.json

```diff
@@ -0,0 +1,3 @@
+{"A": "a", "B": 1}
+{"A": "b", "B": 2}
+{"A": "c", "B": 3}
```

Comment:
data.json file also should contain an empty line on the end (if this is not considered as a test for read_json behavior on a files without a newline in the end)